src/AddIns/Uno.UI.Lottie/LottieVisualSourceBase.cs

```diff
@@ -38,6 +38,18 @@ public Uri UriSource
 			set => SetValue(UriSourceProperty, value);
 		}
 
+		public static DependencyProperty IsAbsoluteProperty { get ; } = DependencyProperty.Register(
```

Comment:
Is it a public property that we missed from Microsoft's implementation?

Reply:
No, it's just something I added, I couldn't even find Microsoft's implementation.
Basically, I don't think there is a way to let the code determine by itself if it is a relative or absolute path so the only idea I had, was to make the user decide.

Reply:
Another idea I just got would have been to change the implementation based on the beginning of the URI. If we check if the URI starts with 'http' then it's absolute if not it's relative, although that feels a bit lazy and probably too specific.

Reply:
Take a look at [this](https://docs.microsoft.com/en-us/dotnet/api/system.uri.trycreate?view=net-6.0). You'll be able to determine if an URI is absolute, and starts with http/https.